docs/integrations/sources/amazon-seller-partner-migrations.md

```diff
@@ -0,0 +1,21 @@
+# Amazon Seller Partner Migration Guide
+
+## Upgrading to 2.0.0
+
+This change removes Brand Analytics from Airbyte Cloud version of connector and permanently removes deprecated FBA reports.
+Customers who have those streams must refresh their schema OR disable next streams:
```

Comment:
Customers who have those streams must refresh their schema OR disable the following streams:

Reply:
done